CHANGES_NEXT_RELEASE

```diff
@@ -17,3 +17,5 @@
 - Add: 'q' expression evaluation in NGSIv2 subscription (at subscription creation/update and update context times) (Issue #1316 and #1658)
 - Fix: subscription service path is lost at update subscription time (Issue #1693)
 - Add: Enable log summary traces using -logSummary CLI (Issue #1585)
+- Fix: Avoiding cases of double BadInput for one and the same request (Issue #1697)
```

Comment:
The first line could be omitted, as is actually covered by: 

" Add: proper alarm management, including activation/deactivation in logs (Issue #1582)"